configure.ac

```diff
@@ -72,6 +72,20 @@ AC_TYPE_UINT8_T
 AC_FUNC_MALLOC
 AC_CHECK_FUNCS([strdup])
 
+LIBVIRT_REQUIRED="1.2.2"
+
+AC_ARG_WITH([libvirt],
+            [AS_HELP_STRING([--without-libvirt],
+                            [run hyperd with libvirt])],
+            [with_libvirt=no],[with_libvirt=yes])
+
+if test "$with_libvirt" = yes; then
+	PKG_CHECK_MODULES([libvirt], [libvirt >= $LIBVIRT_REQUIRED], [], [
+		AC_MSG_ERROR([libvirt-devel >= $LIBVIRT_REQUIRED is required or reconfigure runv with --without-libvirt])])
```

Comment:
--with-xen will be silent turn off when there is no xen-dev installed. I think it should complain and fail.

--with-xen & xen-dev  => xen
--with-xen & no xen-dev  =>  fail
--without-xen & xen-dev  => no xen
--without-xen & no xen-dev  => no xen
xen-dev  => xen
no xen-dev  => no xen

the same for libvirt